build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.4.1'
+        classpath 'com.android.tools.build:gradle:3.5.2'
```

Comment:
Please revert these extra irrelevant changes. You should ignore Android Studio's warnings.

Reply:
As @neslihanturan said, if we want to update the Gradle version, we should create a separate  PR/issue for the same.